student.rb

```diff
@@ -2,6 +2,8 @@
 
 # Defines the Student class
 class Student < Person
+  attr_accessor :classroom
+
   def initialize(classroom)
     super()
     @classroom = classroom
```

Comment:
- [ ] Check for a method that serves as the 'belongs-to' side (a student belongs to a classroom)
```ruby
def classroom=(classroom)
      @classroom = classroom
      classroom.students.push(self) unless classroom.students.include?(self)
  end
```
- [ ] When setting the classroom for a student it also adds it to the classrooms' students: